### DISCUSSION TITLE
[Feature] Drag to select images

### DISCUSSION THREAD

**Discussion by RenaKunisaki** [immich-app/immich#14675] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

I wish I could shift+click and drag a box to select many images at once in the various image-wall views, and Ctrl+click to toggle individual selections, like in most desktop file managers.

### Platform

- [ ] Server
- [X] Web
- [ ] Mobile

**Comment by danieldietzler** [comment]
You can already do that. Select an image, hold shift, and hover over the timeline and you'll see how it multiselects assets

**Comment by bo0tzz** [comment]
https://github.com/user-attachments/assets/6b3974e6-fedc-457e-94c5-fa85573621b0

**Comment by RenaKunisaki** [comment]
It's not working for me in Firefox. It either tries to drag the image (like on most web pages) or does nothing.

**Reply by danieldietzler** [reply]
You don't actually drag. You select one image, then hold shift and hover over the images and click on the last image.

**Reply by RenaKunisaki** [reply]
This is only selecting the images I click on, not any in between.

**Reply by danieldietzler** [reply]
Hold shift while clicking.

**Reply by danieldietzler** [reply]
Have a look at bo0tzz video

**Reply by RenaKunisaki** [reply]
Yes, I've tried holding Shift while clicking the first, after clicking the first, holding it the entire time. I'm not getting the blue highlight shown in the video.

**Reply by danieldietzler** [reply]
Are you on the latest version? Maybe dark reader or some other browser extension that hides it?

**Reply by RenaKunisaki** [reply]
Version is v1.122.2, tried with all extensions disabled. I'll try in Chrome later today.

**Reply by danieldietzler** [reply]
To clarify, bo0tzz and I both use firefox and it works, so not sure what's going on over there

**Reply by RenaKunisaki** [reply]
Sorry for the delay. It is working in Chrome. Firefox just seems to act like Shift isn't held.
